### PR TITLE
Separate Circle from CircleMarker

### LINKED ISSUE #339: Circle radius units
The folium documentation for a circle says that the radius parameter is "Circle radius in pixels"

http://folium.readthedocs.org/en/latest/#circle-markers

The live example shows that it's calling L.circle() JavaScript method and passing the radius to that method.

Jumping over to Leaflet documentation, it says that radius is "... radius of a circle. Units are in meters."

http://leafletjs.com/reference.html#circle

So isn't the radius passed to folium actually the radius in meters? If that's the case, I can submit a PR to update the documentation. Otherwise, am I missing a conversion somewhere?

### PR DESCRIPTION
**Pull request by BibMartin**
Fix #339.

BTW, this change is required for https://github.com/ocefpaf/folium_notebooks/pull/4 to work.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Need some test template update to pass.

**Comment by BibMartin** [comment]
Yes, I'll have a look to it in the coming hours.

File: folium/features.py [hunk 1]
```diff
@@ -634,6 +634,51 @@ def __init__(self, html=None, icon_size=None, icon_anchor=None,
...
+    location: tuple or list, default None
+        Latitude and Longitude of Marker (Northing, Easting)
+    radius: int
+        The radius of the circle in meters. For setting the radius in pixel,
+        please use CircleMarker.
+    color: str, default 'black'
+        The color of the marker's edge in a HTML-compatible format.
+    fill_color: str, default 'black'
+        The fill color of the marker in a HTML-compatible format.
+    fill_opacity: float, default à.6
```
Comment (ocefpaf) [inline folium/features.py]: Non-english keyboards and its pitfalls `à.6` :wink:

File: folium/features.py [hunk 2]
```diff
@@ -634,6 +634,51 @@ def __init__(self, html=None, icon_size=None, icon_anchor=None,
...
+class Circle(Marker):
+    """Creates a Circle object for plotting on a Map.
+
+    Parameters
+    ----------
+    location: tuple or list, default None
+        Latitude and Longitude of Marker (Northing, Easting)
+    radius: int
+        The radius of the circle in meters. For setting the radius in pixel,
+        please use CircleMarker.
```
Comment (ocefpaf) [inline folium/features.py]: ~~please~~ use CircleMarker (no need to say please in the docs. Unless it is please read the docs :stuck_out_tongue_winking_eye:)

File: folium/features.py [hunk 3]
```diff
@@ -642,7 +687,8 @@ class CircleMarker(Marker):
     location: tuple or list, default None
         Latitude and Longitude of Marker (Northing, Easting)
     radius: int
-        The radius of the circle in pixels.
+        The radius of the circle in pixels. For setting the radius in meter,
+        please use Circle.
```
Comment (ocefpaf) [inline folium/features.py]: Same as https://github.com/python-visualization/folium/pull/447/files#r66618882

**Comment by ocefpaf** [comment]
No rush @BibMartin. I also have tons of work and I am lurking here to avoid facing it :unamused:

**Comment by BibMartin** [comment]
Fixed and squashed !

**Comment by ocefpaf** [comment]
> Fixed and squashed !

and merged!

**Pull request merged**